File: ClientApp/ClientApp/Models/Responses/GetPlayersInRoomResponse.cs
using ClientApp.Models.ResponseErrors;
using Newtonsoft.Json;
namespace ClientApp.Models.Responses
{
    class GetPlayersInRoomResponse : Response<NoResponseErrors>
    {
        private GetPlayersInRoomResponse() { }

        [JsonProperty]
        public List<LoggedUser> Players { get; private set; }
    }
}

File: M_Trivia/M_Trivia/GetPlayersInRoomResponse.cpp
#include "GetPlayersInRoomResponse.hpp"
#include "json.hpp"


GetPlayersInRoomResponse::GetPlayersInRoomResponse(unsigned int status, const std::vector<LoggedUser>& players) : Response(status), players(players) {

}


nlohmann::json GetPlayersInRoomResponse::getJson() const {
        nlohmann::json j = Response::getJson();
        j["Players"] = players; 
        return j;
    }


